### ISSUE TITLE
android app - update tiles does not work - quick tiles still "not setup"

### ISSUE THREAD

**Issue by GitJamz** [home-assistant/android#2036]
Hi,

I'm using homeassisant android app from f-droid.

Everything works well, even widgets to call scripts.
But to set up a quick tile does not work. I click the update tile data button and it is successful in the app, but the tiles are still shown as "tile not setup".

Tried rebooting phone etc.. any advice?

thanks

**Comment by dshokouhi** [comment]
This issue is missing the issue template that gets presented when a new issue is created. A lot of important information is missing  here, please copy the template from a new issue now that you are logged into github and fill it out here so we can get a clear picture.

https://github.com/home-assistant/android/issues/new/choose


Are you certain you updated the correct tile? There are a total of 12, try to select the first tile and set that up then drag the first one from the list. You have to be precise with the tile you drag over and the one you use when setting it up in app settings. If that fails then we need the logs as requested in the template.

**Comment by GitJamz** [comment]
> Are you certain you updated the correct tile? There are a total of 12, try to select the first tile and set that up then drag the first one from the list. You have to be precise with the tile you drag over and the one you use when setting it up in app settings. If that fails then we need the logs as requested in the template.

Hey, thanks for quick reply.
Yes I update the correct tile that I just edited.. Ive tried several times.
Are there some sensors or permissions that homeassistant app needs to have access to for this to work?

**Comment by dshokouhi** [comment]
> Are there some sensors or permissions that homeassistant app needs to have access to for this to work?

you should first get the issue template filled out including the logs before we can continue

**Comment by rollintoy** [comment]
I'm seeing same issue. Not able to gather logs at moment. I'm on Pixel 5 with latest updates and beta companion app. I'm able to see the updated tile in the app but despite clicking "update...." the tile never updates.

**Comment by dshokouhi** [comment]
Logs will be critical here to see if there is a failure as I still cannot reproduce this.  Make sure you guys are setting the tile data for the correct tile. I know it can be confusing to have 12 tiles with the same default name. Try to either select the first tile in the list of HA tiles or select the very last one and then ensure you set the data for the correct tile in the "List of Tiles" drop down. The order you pick and select from that drop down is important.  

Also it would be great if someone can provide the issue template while also ensuring they are on the latest app version. There is still important data missing here.

**Comment by dshokouhi** [comment]
Just a FYI the latest beta build of the android app which is being pushed to the play store tonight has some new debug logs specifically meant to show the tile being saved in the app and also the tile that was dragged/clicked in the quick settings area. If possible please test with the latest beta and get us the logs if the tile ID matches.

**Comment by x-plood** [comment]
I have a same problem.

**Home Assistant Android version:**

2022.1.1-full

**Android version:**
11

**Phone model:**
Samsung Galaxy s10+

**Home Assistant version:**
Home Assistant 2021.12.1

<details>
<summary>Log</summary>
`01-06 13:21:43.764 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: MSG_WINDOW_FOCUS_CHANGED 1 1
01-06 13:21:43.765 11236 11236 D InputMethodManager: prepareNavigationBarInfo() DecorView@8b65d11[SettingsActivity]
01-06 13:21:43.765 11236 11236 D InputMethodManager: getNavigationBarColor() -15658735
01-06 13:21:43.767 11236 11236 D InputMethodManager: prepareNavigationBarInfo() DecorView@8b65d11[SettingsActivity]
01-06 13:21:43.767 11236 11236 D InputMethodManager: getNavigationBarColor() -15658735
01-06 13:21:43.767 11236 11236 V InputMethodManager: Starting input: tba=io.homeassistant.companion.android ic=null mNaviBarColor -15658735 mIsGetNaviBarColorSuccess true , NavVisible : true , NavTrans : false
01-06 13:21:43.767 11236 11236 D InputMethodManager: startInputInner - Id : 0
01-06 13:21:43.767 11236 11236 I InputMethodManager: startInputInner - mService.startInputOrWindowGainedFocus
01-06 13:21:43.770 11236 11236 D InputTransport: Input channel constructed: 'ClientS', fd=178
01-06 13:21:43.770 11236 11236 D InputTransport: Input channel destroyed: 'ClientS', fd=182
01-06 13:21:44.222 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: MSG_WINDOW_FOCUS_CHANGED 0 1
01-06 13:21:44.401 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866462752]
01-06 13:21:44.401 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866462752]
01-06 13:21:44.401 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866464096]
01-06 13:21:44.401 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866464096]
01-06 13:21:44.401 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866462416]
01-06 13:21:44.401 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866462416]
01-06 13:21:44.451 11236 11236 D InputTransport: Input channel destroyed: 'ClientS', fd=178
01-06 13:21:44.468 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866464880]
01-06 13:21:44.468 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866464880]
01-06 13:21:47.148 11236 12305 W libEGL  : EGLNativeWindowType 0x6ec1d5bd70 disconnect failed
01-06 13:21:47.163 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866485824]
01-06 13:21:47.163 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866485824]
01-06 13:21:47.164 11236 11236 I SurfaceControl: assignNativeObject: nativeObject = 0 Surface(name=null)/@0xace3e32 / android.view.SurfaceControl.readFromParcel:1117 android.view.IWindowSession$Stub$Proxy.relayout:1810 android.view.ViewRootImpl.relayoutWindow:9005 android.view.ViewRootImpl.performTraversals:3360 android.view.ViewRootImpl.doTraversal:2618 android.view.ViewRootImpl$TraversalRunnable.run:9971 android.view.Choreographer$CallbackRecord.run:1010 android.view.Choreographer.doCallbacks:809 android.view.Choreographer.doFrame:744 android.view.Choreographer$FrameDisplayEventReceiver.run:995 
01-06 13:21:47.164 11236 11236 I SurfaceControl: assignNativeObject: nativeObject = 0 Surface(name=null)/@0x1da3912 / android.view.SurfaceControl.readFromParcel:1117 android.view.IWindowSession$Stub$Proxy.relayout:1820 android.view.ViewRootImpl.relayoutWindow:9005 android.view.ViewRootImpl.performTraversals:3360 android.view.ViewRootImpl.doTraversal:2618 android.view.ViewRootImpl$TraversalRunnable.run:9971 android.view.Choreographer$CallbackRecord.run:1010 android.view.Choreographer.doCallbacks:809 android.view.Choreographer.doFrame:744 android.view.Choreographer$FrameDisplayEventReceiver.run:995 
01-06 13:21:47.168 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: Relayout returned: old=(0,0,1440,3040) new=(0,0,1440,3040) req=(1440,3040)8 dur=15 res=0x5 s={false 0} ch=true fn=572
01-06 13:21:47.185 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: stopped(true) old=false
01-06 13:21:47.199 11236 11236 I SurfaceControl: assignNativeObject: nativeObject = 0 Surface(name=null)/@0xace3e32 / android.view.SurfaceControl.readFromParcel:1117 android.view.IWindowSession$Stub$Proxy.relayout:1810 android.view.ViewRootImpl.relayoutWindow:9005 android.view.ViewRootImpl.performTraversals:3360 android.view.ViewRootImpl.doTraversal:2618 android.view.ViewRootImpl$TraversalRunnable.run:9971 android.view.Choreographer$CallbackRecord.run:1010 android.view.Choreographer.doCallbacks:809 android.view.Choreographer.doFrame:744 android.view.Choreographer$FrameDisplayEventReceiver.run:995 
01-06 13:21:47.200 11236 11236 I SurfaceControl: assignNativeObject: nativeObject = 0 Surface(name=null)/@0x1da3912 / android.view.SurfaceControl.readFromParcel:1117 android.view.IWindowSession$Stub$Proxy.relayout:1820 android.view.ViewRootImpl.relayoutWindow:9005 android.view.ViewRootImpl.performTraversals:3360 android.view.ViewRootImpl.doTraversal:2618 android.view.ViewRootImpl$TraversalRunnable.run:9971 android.view.Choreographer$CallbackRecord.run:1010 android.view.Choreographer.doCallbacks:809 android.view.Choreographer.doFrame:744 android.view.Choreographer$FrameDisplayEventReceiver.run:995 
01-06 13:21:47.201 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: Relayout returned: old=(0,0,1440,3040) new=(0,0,1440,3040) req=(1440,3040)8 dur=6 res=0x5 s={false 0} ch=false fn=-1
01-06 13:21:47.540 11236 11307 D WebSocketRepository: Sending message 30: {type=config/area_registry/list, id=30}
01-06 13:21:47.541 11236 11307 D WebSocketRepository: Message number 30 sent
01-06 13:21:47.572 11236 11250 I mpanion.androi: Background concurrent copying GC freed 184975(8665KB) AllocSpace objects, 10(732KB) LOS objects, 45% free, 28MB/52MB, paused 152us total 386.926ms
01-06 13:21:47.589 11236 11388 D WebSocketRepository: Websocket: onMessage (text)
01-06 13:21:47.589 11236 11388 D WebSocketRepository: Message number 30 received: {"id": 30, "type": "result", "success": true, "result": [{"area_id": "gostinaia", "name": "\u0413\u043e\u0441\u0442\u0438\u043d\u0430\u044f", "picture": null}, {"area_id": "kukhnia", "name": "\u041a\u0443\u0445\u043d\u044f", "picture": null}, {"area_id": "spalnia", "name": "\u0421\u043f\u0430\u043b\u044c\u043d\u044f", "picture": null}, {"area_id": "koridor", "name": "\u041a\u043e\u0440\u0438\u0434\u043e\u0440", "picture": null}, {"area_id": "dacha", "name": "\u0414\u0430\u0447\u0430", "picture": null}]}
01-06 13:21:47.590 11236 11307 D WebSocketRepository: Sending message 31: {type=config/device_registry/list, id=31}
01-06 13:21:47.590 11236 11307 D WebSocketRepository: Message number 31 sent
01-06 13:21:47.661 11236 11388 D WebSocketRepository: Websocket: onMessage (text)
01-06 13:21:47.663 11236 11388 D WebSocketRepository: Message number 31 received: {"id": 31, "type": "result", "success": true, "result": [{"area_id": null, "configuration_url": "https://www.met.no/en", "config_entries": ["ba1c02e5a4cea224bbb896314546426c"], "connections": [], "disabled_by": null, "entry_type": "service", "id": "0aafa9b411b2b624ee69294099205c31", "identifiers": [["met"]], "manufacturer": "Met.no", "model": "Forecast", "name_by_user": null, "name": "Forecast", "sw_version": null, "via_device_id": null}, {"area_id": "koridor", "configuration_url": "http://192.168.113.1", "config_entries": ["db4543c8a97b887e98c4648e0f1db922"], "connections": [["upnp", "uuid:UUID-MIKROTIK-INTERNET-GATEWAY-DEVICE-V8RJ-QEH6"]], "disabled_by": null, "entry_type": null, "id": "f2f8ff3f05b49071979bce76a7b12522", "identifiers": [["upnp", "uuid:UUID-MIKROTIK-INTERNET-GATEWAY-DEVICE-V8RJ-QEH6"]], "manufacturer": "MikroTik", "model": "Router OS", "name_by_user": null, "name": "MikroTik Router", "sw_version": null, "via_device_id": null}, {"area_id": null, "configuration_url": null, "config_entries": ["3804d73ccaab72a6e8db7c0d9662f773"], "connections": [], "disabled_by": null, "entry_type": null, "id": "76411cca0a2a53be2737a59f85b80f11", "identifiers": [["mobile_app", "a058fbd42d2499dc"]], "manufacturer": "samsung", "model": "SM-G975F", "name_by_user": null, "name": "ploodS10+", "sw_version": "30", "via_device_id": null}, {"area_id": "spalnia", "configuration_url": null, "config_entries": ["cc42f886ad8173e3b8fad27649094a99"], "connections": [], "disabled_by": null, "entry_type": null, "id": "9c968599b1941b5356ceac88a5fcb8e6", "identifiers": [["kodi", "cc42f886ad8173e3b8fad27649094a99"]], "manufacturer": "Kodi", "model": null, "name_by_user": null, "name": null, "sw_version": "18.9", "via_device_id": null}, {"area_id": "kukhnia", "configuration_url": null, "config_entries": ["69ac623328d5b6a4605bbd4071a6be06"], "connections": [], "disabled_by": null, "entry_type": null, "id": "79e44eb4fde1f5c2af5f6ac658725f38", "identifiers": [["kodi", "290da4de-f812-4cd0-a4a5-ee44e92c9bac"]], "manufacturer": "Kodi", "model": null, "name_by_user": null, "name": "Android", "sw_version": "19.3", "via_device_id": null}, {"area_id": "spalnia", "configuration_url": null, "config_entries": ["c571ba2a2d56e1de02489bb2a631373e"], "connections": [], "disabled_by": null, "entry_type": null, "id": "38c39170fa0d27cab1549bc45c98b76f", "identifiers": [["ipp", "00000000-0000-1000-8000-00183b165790"]], "manufacturer": "Canon", "model": "G3010 series", "name_by_user": null, "name": "Canon G3010 series", "sw_version": "1.070", "via_device_id": null}, {"area_id": "kukhnia", "configuration_url": null, "config_entries": ["660c897146274e2ccbe257c9d15aec9f"], "connections": [], "disabled_by": null, "entry_type": null, "id": "d09503ecaef70c7051aa04398964eb55", "identifiers": [["cast", "1b8d5961431c4e3fab5558c9e952dfcc"]], "manufacturer": "None", "model": "AirReceiver", "name_by_user": null, "name": "Kuhnia", "sw_version": null, "via_device_id": null}, {"area_id": null, "configuration_url": null, "config_entries": ["2354e61e9eab00807090cd761bb7bc9b"], "connections": [], "disabled_by": null, "entry_type": null, "id": "542bf80a45025d2a66cec4f7be5881b1", "identifiers": [["mobile_app", "80396d70a78cc00a"]], "manufacturer": "samsung", "model": "SM-G973F", "name_by_user": null, "name": "HeoniaS10", "sw_version": "30", "via_device_id": null}, {"area_id": null, "configuration_url": null, "config_entries": ["fed5223db6a66e5d2c74166549066f85"], "connections": [], "disabled_by": null, "entry_type": "service", "id": "0abbc68a48988915c80b434c11cafc3f", "identifiers": [["hacs", "0717a0cd-745c-48fd-9b16-c8534c9704f9-bc944b0f-fd42-4a58-a072-ade38d1444cd"]], "manufacturer": "hacs.xyz", "model": "", "name_by_user": null, "name": "HACS", "sw_version": "1.13.2", "via_device_id": null}, {"area_id": "spalnia", "configuration_url": null, "config_entries": ["f606201972466a9ea65b8b785b4c157d"], "connections": [], "disabled_by": null, "entry_type": null, "id": "90008118bcd231bec20ae89a953d906a", "identifier
01-06 13:21:47.669 11236 11307 E HaConProService: Error getting list of entities
01-06 13:21:47.669 11236 11307 E HaConProService: java.lang.IllegalArgumentException: Cannot construct instance of `java.util.ArrayList` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('Mikrotik Konkovo ax RB1100x4')
01-06 13:21:47.669 11236 11307 E HaConProService:  at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: java.util.ArrayList[73]->io.homeassistant.companion.android.common.data.websocket.impl.entities.DeviceRegistryResponse["identifiers"]->java.util.ArrayList[0])
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4393)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.ObjectMapper.convertValue(ObjectMapper.java:4334)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at io.homeassistant.companion.android.common.data.websocket.impl.WebSocketRepositoryImpl.getDeviceRegistry(WebSocketRepositoryImpl.kt:386)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at io.homeassistant.companion.android.common.data.websocket.impl.WebSocketRepositoryImpl$getDeviceRegistry$1.invokeSuspend(Unknown Source:14)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at kotlinx.coroutines.internal.ScopeCoroutine.afterResume(Scopes.kt:33)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at kotlinx.coroutines.internal.LimitedDispatcher.run(LimitedDispatcher.kt:39)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:95)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
01-06 13:21:47.669 11236 11307 E HaConProService: Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `java.util.ArrayList` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('Mikrotik Konkovo ax RB1100x4')
01-06 13:21:47.669 11236 11307 E HaConProService:  at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: java.util.ArrayList[73]->io.homeassistant.companion.android.common.data.websocket.impl.entities.DeviceRegistryResponse["identifiers"]->java.util.ArrayList[0])
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1728)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.DeserializationContext.handleMissingInstantiator(DeserializationContext.java:1353)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.std.StdDeserializer._deserializeFromString(StdDeserializer.java:311)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.std.StringCollectionDeserializer.handleNonArray(StringCollectionDeserializer.java:284)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.std.StringCollectionDeserializer.deserialize(StringCollectionDeserializer.java:192)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.std.StringCollectionDeserializer.deserialize(StringCollectionDeserializer.java:182)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.std.StringCollectionDeserializer.deserialize(StringCollectionDeserializer.java:25)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer._deserializeFromArray(CollectionDeserializer.java:355)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:244)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:28)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:542)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeWithErrorWrapping(BeanDeserializer.java:563)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:438)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1405)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:351)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:184)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer._deserializeFromArray(CollectionDeserializer.java:355)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:244)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:28)
01-06 13:21:47.669 11236 11307 E HaConProService: 	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4388)
01-06 13:21:47.669 11236 11307 E HaConProService: 	... 14 more
01-06 13:21:50.796 11236 11236 I SurfaceControl: assignNativeObject: nativeObject = 0 Surface(name=null)/@0xace3e32 / android.view.SurfaceControl.readFromParcel:1117 android.view.IWindowSession$Stub$Proxy.relayout:1810 android.view.ViewRootImpl.relayoutWindow:9005 android.view.ViewRootImpl.performTraversals:3360 android.view.ViewRootImpl.doTraversal:2618 android.view.ViewRootImpl$TraversalRunnable.run:9971 android.view.Choreographer$CallbackRecord.run:1010 android.view.Choreographer.doCallbacks:809 android.view.Choreographer.doFrame:744 android.view.Choreographer$FrameDisplayEventReceiver.run:995 
01-06 13:21:50.796 11236 11236 I SurfaceControl: assignNativeObject: nativeObject = 0 Surface(name=null)/@0x1da3912 / android.view.SurfaceControl.readFromParcel:1117 android.view.IWindowSession$Stub$Proxy.relayout:1820 android.view.ViewRootImpl.relayoutWindow:9005 android.view.ViewRootImpl.performTraversals:3360 android.view.ViewRootImpl.doTraversal:2618 android.view.ViewRootImpl$TraversalRunnable.run:9971 android.view.Choreographer$CallbackRecord.run:1010 android.view.Choreographer.doCallbacks:809 android.view.Choreographer.doFrame:744 android.view.Choreographer$FrameDisplayEventReceiver.run:995 
01-06 13:21:50.797 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: Relayout returned: old=(0,0,1440,3040) new=(0,0,1440,3040) req=(1440,3040)4 dur=7 res=0x1 s={false 0} ch=false fn=-1
01-06 13:21:50.797 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: stopped(false) old=true
01-06 13:21:50.815 11236 11236 I SurfaceControl: assignNativeObject: nativeObject = 0 Surface(name=null)/@0x1da3912 / android.view.SurfaceControl.readFromParcel:1117 android.view.IWindowSession$Stub$Proxy.relayout:1820 android.view.ViewRootImpl.relayoutWindow:9005 android.view.ViewRootImpl.performTraversals:3360 android.view.ViewRootImpl.doTraversal:2618 android.view.ViewRootImpl$TraversalRunnable.run:9971 android.view.Choreographer$CallbackRecord.run:1010 android.view.Choreographer.doCallbacks:809 android.view.Choreographer.doFrame:744 android.view.Choreographer$FrameDisplayEventReceiver.run:995 
01-06 13:21:50.817 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: Relayout returned: old=(0,0,1440,3040) new=(0,0,1440,3040) req=(1440,3040)0 dur=8 res=0x7 s={true 475698412896} ch=true fn=-1
01-06 13:21:50.818 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: [DP] dp(1) 1 android.view.ViewRootImpl.reportNextDraw:10957 android.view.ViewRootImpl.performTraversals:3845 android.view.ViewRootImpl.doTraversal:2618 
01-06 13:21:50.818 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: [DP] pd() Asnyc report
01-06 13:21:50.846 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: [DP] pdf(0) 1 android.view.ViewRootImpl.lambda$performDraw$1$ViewRootImpl:4668 android.view.-$$Lambda$ViewRootImpl$DJd0VUYJgsebcnSohO6h8zc_ONI.run:6 android.os.Handler.handleCallback:938 
01-06 13:21:50.846 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: [DP] rdf()
01-06 13:21:51.720 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: stopped(false) old=false
01-06 13:21:52.154 11236 11236 D WebSocketRepository: Sending message 32: {type=get_states, id=32}
01-06 13:21:52.155 11236 11236 D WebSocketRepository: Message number 32 sent
01-06 13:21:52.345 11236 11388 D WebSocketRepository: Websocket: onMessage (text)
01-06 13:21:52.354 11236 11388 D WebSocketRepository: Message number 32 received: {"id": 32, "type": "result", "success": true, "result": [{"entity_id": "person.plood", "state": "\u041d\u0430 \u0440\u0430\u0431\u043e\u0442\u0435", "attributes": {"editable": false, "id": "plood", "latitude": 55.8232355, "longitude": 37.6091399, "gps_accuracy": 16, "source": "device_tracker.plood_s10_2", "user_id": "f1a082fb4cd94c29afed711d5537d43e", "entity_picture": "/api/image/serve/162b819acbed27488292e401d5013c21/512x512", "friendly_name": "\u0412\u043e\u0432\u0430"}, "last_changed": "2022-01-06T08:56:17.514187+00:00", "last_updated": "2022-01-06T10:07:51.663489+00:00", "context": {"id": "ecfd42ef8434c9658de27449d15f3924", "parent_id": null, "user_id": null}}, {"entity_id": "person.heonia", "state": "home", "attributes": {"editable": false, "id": "heonia", "source": "device_tracker.heonia_samsung_s10", "user_id": "0ee4c4098620484d9d0081dd4753b6af", "entity_picture": "/api/image/serve/6a539295f34ae2e11eaa03df2e2f9c29/512x512", "friendly_name": "\u0422\u0443\u0441\u044f"}, "last_changed": "2022-01-05T20:57:59.391851+00:00", "last_updated": "2022-01-05T20:58:18.938804+00:00", "context": {"id": "7c16e9385af6cc2f06c7996d4dc7be7d", "parent_id": null, "user_id": null}}, {"entity_id": "sun.sun", "state": "above_horizon", "attributes": {"next_dawn": "2022-01-07T05:08:52.196923+00:00", "next_dusk": "2022-01-06T14:02:24.712962+00:00", "next_midnight": "2022-01-06T21:36:09+00:00", "next_noon": "2022-01-07T09:35:57+00:00", "next_rising": "2022-01-07T05:54:38.838814+00:00", "next_setting": "2022-01-06T13:16:27.681189+00:00", "elevation": 11.42, "azimuth": 190.31, "rising": false, "friendly_name": "Sun"}, "last_changed": "2022-01-06T05:55:14.140490+00:00", "last_updated": "2022-01-06T10:19:31.013870+00:00", "context": {"id": "d948bc5b27f9980301f5857ea5736298", "parent_id": null, "user_id": null}}, {"entity_id": "scene.spalnia_side_light_on", "state": "scening", "attributes": {"entity_id": ["switch.udlinitel_rozetka_1", "switch.udlinitel_rozetka_2", "switch.udlinitel_rozetka_4"], "friendly_name": "Spalnia side light on"}, "last_changed": "2022-01-05T20:58:00.496724+00:00", "last_updated": "2022-01-05T20:58:00.496724+00:00", "context": {"id": "b6fc5c15b43912192cbfed81dd8f0eb6", "parent_id": null, "user_id": null}}, {"entity_id": "scene.spalnia_side_light_off", "state": "scening", "attributes": {"entity_id": ["switch.udlinitel_rozetka_1", "switch.udlinitel_rozetka_2", "switch.udlinitel_rozetka_4"], "friendly_name": "Spalnia side light off"}, "last_changed": "2022-01-05T20:58:00.496881+00:00", "last_updated": "2022-01-05T20:58:00.496881+00:00", "context": {"id": "085af0766c991fd7c9074ceec0a1ec82", "parent_id": null, "user_id": null}}, {"entity_id": "scene.evening_light", "state": "scening", "attributes": {"entity_id": ["switch.udlinitel_rozetka_1", "switch.sonoff_usb1", "switch.udlinitel_rozetka_2", "switch.udlinitel_rozetka_4", "switch.liustra_v_spalne"], "friendly_name": "Evening light"}, "last_changed": "2022-01-05T20:58:00.497014+00:00", "last_updated": "2022-01-05T20:58:00.497014+00:00", "context": {"id": "50b0fb578440e3d362c2933f11bea6b8", "parent_id": null, "user_id": null}}, {"entity_id": "input_number.airhumidifier_target_humidity", "state": "54.0", "attributes": {"initial": null, "editable": false, "min": 1.0, "max": 99.0, "step": 1.0, "mode": "slider", "unit_of_measurement": "%", "friendly_name": "Target humidity"}, "last_changed": "2022-01-05T20:58:00.688576+00:00", "last_updated": "2022-01-05T20:58:00.688576+00:00", "context": {"id": "fd6a257670716c96c9b8bbb2364a5502", "parent_id": null, "user_id": null}}, {"entity_id": "input_number.giran_volume_set", "state": "60.0", "attributes": {"initial": null, "editable": false, "min": 0.0, "max": 100.0, "step": 1.0, "mode": "slider", "friendly_name": "\u0413\u0440\u043e\u043c\u043a\u043e\u0441\u0442\u044c \u043a\u043e\u043c\u043f\u044c\u044e\u0442\u0435\u0440\u0430"}, "last_changed": "2022-01-05T20:58:00.688669+00:00", "last_updated": "2022-01-05T20:58:00.688669+00:00", "context": {"id": "4fbe40ff4e9d97
01-06 13:21:52.377 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866245808]
01-06 13:21:52.377 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866245808]
01-06 13:21:52.377 11236 11236 I SurfaceControl: assignNativeObject: nativeObject = 0 Surface(name=null)/@0x1da3912 / android.view.SurfaceControl.readFromParcel:1117 android.view.IWindowSession$Stub$Proxy.relayout:1820 android.view.ViewRootImpl.relayoutWindow:9005 android.view.ViewRootImpl.performTraversals:3360 android.view.ViewRootImpl.doTraversal:2618 android.view.ViewRootImpl$TraversalRunnable.run:9971 android.view.Choreographer$CallbackRecord.run:1010 android.view.Choreographer.doCallbacks:809 android.view.Choreographer.doFrame:744 android.view.Choreographer$FrameDisplayEventReceiver.run:995 
01-06 13:21:52.378 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866247264]
01-06 13:21:52.378 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866247264]
01-06 13:21:52.378 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866253088]
01-06 13:21:52.378 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866253088]
01-06 13:21:52.378 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: Relayout returned: old=(0,0,1440,3040) new=(0,0,1440,3040) req=(1440,3040)0 dur=2 res=0x1 s={true 475698412896} ch=false fn=3
01-06 13:21:52.382 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: MSG_WINDOW_FOCUS_CHANGED 1 1
01-06 13:21:52.382 11236 11236 D InputMethodManager: prepareNavigationBarInfo() DecorView@8b65d11[SettingsActivity]
01-06 13:21:52.382 11236 11236 D InputMethodManager: getNavigationBarColor() -15658735
01-06 13:21:52.383 11236 11236 D InputMethodManager: prepareNavigationBarInfo() DecorView@8b65d11[SettingsActivity]
01-06 13:21:52.383 11236 11236 D InputMethodManager: getNavigationBarColor() -15658735
01-06 13:21:52.383 11236 11236 V InputMethodManager: Starting input: tba=io.homeassistant.companion.android ic=null mNaviBarColor -15658735 mIsGetNaviBarColorSuccess true , NavVisible : true , NavTrans : false
01-06 13:21:52.383 11236 11236 D InputMethodManager: startInputInner - Id : 0
01-06 13:21:52.383 11236 11236 I InputMethodManager: startInputInner - mService.startInputOrWindowGainedFocus
01-06 13:21:52.385 11236 11236 D InputTransport: Input channel constructed: 'ClientS', fd=182
01-06 13:21:52.399 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866505984]
01-06 13:21:52.399 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866505984]
01-06 13:21:52.399 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866245808]
01-06 13:21:52.399 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866245808]
01-06 13:21:53.020 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: ViewPostIme key 0
01-06 13:21:53.023 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: ViewPostIme key 1
01-06 13:21:53.047 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866505760]
01-06 13:21:53.047 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866505760]
01-06 13:21:53.048 11236 11236 I SurfaceControl: assignNativeObject: nativeObject = 0 Surface(name=null)/@0x1da3912 / android.view.SurfaceControl.readFromParcel:1117 android.view.IWindowSession$Stub$Proxy.relayout:1820 android.view.ViewRootImpl.relayoutWindow:9005 android.view.ViewRootImpl.performTraversals:3360 android.view.ViewRootImpl.doTraversal:2618 android.view.ViewRootImpl$TraversalRunnable.run:9971 android.view.Choreographer$CallbackRecord.run:1010 android.view.Choreographer.doCallbacks:809 android.view.Choreographer.doFrame:744 android.view.Choreographer$FrameDisplayEventReceiver.run:995 
01-06 13:21:53.049 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866461408]
01-06 13:21:53.049 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866461408]
01-06 13:21:53.049 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866460064]
01-06 13:21:53.049 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866460064]
01-06 13:21:53.049 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866461296]
01-06 13:21:53.049 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866461296]
01-06 13:21:53.049 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: Relayout returned: old=(0,0,1440,3040) new=(0,0,1440,3040) req=(1440,3040)0 dur=6 res=0x1 s={true 475698412896} ch=false fn=6
01-06 13:21:55.673 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: ViewPostIme pointer 0
01-06 13:21:55.764 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: ViewPostIme pointer 1
01-06 13:21:55.803 11236 11236 D ScrollView: initGoToTop
01-06 13:21:55.848 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866509680]
01-06 13:21:55.848 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866509680]
01-06 13:21:55.849 11236 11236 I SurfaceControl: assignNativeObject: nativeObject = 0 Surface(name=null)/@0x1da3912 / android.view.SurfaceControl.readFromParcel:1117 android.view.IWindowSession$Stub$Proxy.relayout:1820 android.view.ViewRootImpl.relayoutWindow:9005 android.view.ViewRootImpl.performTraversals:3360 android.view.ViewRootImpl.doTraversal:2618 android.view.ViewRootImpl$TraversalRunnable.run:9971 android.view.Choreographer$CallbackRecord.run:1010 android.view.Choreographer.doCallbacks:809 android.view.Choreographer.doFrame:744 android.view.Choreographer$FrameDisplayEventReceiver.run:995 
01-06 13:21:55.849 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866512480]
01-06 13:21:55.849 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866512480]
01-06 13:21:55.849 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866511136]
01-06 13:21:55.849 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866511136]
01-06 13:21:55.850 11236 11236 I SurfaceControl: nativeRelease nativeObject s[470866505760]
01-06 13:21:55.850 11236 11236 I SurfaceControl: nativeRelease nativeObject e[470866505760]
01-06 13:21:55.850 11236 11236 I ViewRootImpl@c78fa86[SettingsActivity]: Relayout returned: old=(0,0,1440,3040) new=(0,0,1440,3040) req=(1440,3040)0 dur=6 res=0x1 s={true 475698412896} ch=false fn=29
01-06 13:21:55.855 11236 11500 D LogcatReader: Read logcat for pid 11236
`
</details>

I hope this is what you need

**Comment by dshokouhi** [comment]
@x-plood  this is unrelated to this issue. Your issue is about Device Controls, this issue is about Quick Setting Tiles.

**Comment by x-plood** [comment]
> @x-plood this is unrelated to this issue. Your issue is about Device Controls, this issue is about Quick Setting Tiles.

I'm sorry I thought it was the same problem

**Comment by prantzos** [comment]
Hey people!

I am having the same issue. Just tried today to set up a few quick settings for some scenes and alarm automations I have configured, but my quick settings toggles are like 

![computer-says-no-small](https://user-images.githubusercontent.com/8318629/150682742-e37a84fa-6f47-4d2d-b105-879e8ccdbace.gif)


So here is any information you might need according to your [bug template](https://github.com/home-assistant/android/issues/new?labels=bug&template=Bug_report.md)

**Home Assistant Android version:** `2022.1.1-full`
**Android version:** `12`
**Android security update:** `January 5, 2020`
**Google Play system update:** `December 1, 2021`
**Kernel version:** `4.19.202-g76868f17c3ad-ab7943297`
**Build number:** `SQ1A.220105.002`
**Phone model:** `Pixel 5`
**Home Assistant version:** `core-2021.12.10`
**Host Operating System:** `Home Assistant OS 7.1`
**Supervisor Version:** `supervisor-2021.12.2`

**Last working Home Assistant release (if known):** 🤷‍♂️ 😁

**Description of problem:** After configuring a quick settings tile and updating the tile data, all 12 quick settings tiles still appear as `Tile Not Setup`

**Traceback (if applicable, to get the logs you may refer to: https://companion.home-assistant.io/docs/troubleshooting/faqs/#android-crash-logs):**
[homeassistant_companion_log_0-23-2022_15-0-35.txt](https://github.com/home-assistant/android/files/7920635/homeassistant_companion_log_0-23-2022_15-0-35.txt)

**Screenshot of problem:** 

![Screenshot_20220123-145845](https://user-images.githubusercontent.com/8318629/150682266-5ac58edc-4a1e-4bef-b066-f0d150e624f6.png)
![Screenshot_20220123-145912](https://user-images.githubusercontent.com/8318629/150682268-9d8c36c8-b212-401e-b0f3-ae7ca92a6128.png)

**Comment by dshokouhi** [comment]
> After configuring a quick settings tile and updating the tile data, all 12 quick settings tiles still appear as Tile Not Setup

Logs indicate you forgot to actually add the tile. After you edit them you are supposed to drag them to the active area.

**Comment by prantzos** [comment]
@dshokouhi I am a bit confused.. After I edit a Tile in `Manage Tiles`, drag what and where? I do not think I can drag an item from that Android Activity to the top of the screen, expanding the notifications and showing the active tiles.. 

I think you must be referring to the part in the [documentation](https://companion.home-assistant.io/docs/integrations/android-quick-settings/) where it is mentioned that: 
_**Once you have updated your tile data you will be able to edit your devices quick settings menu and then you can drag the Home Assistant icon from the list of tiles into the active section**_  

As you can see from my second screenshot `ALL 12`available HA tiles are still named `Tile Not Setup` and none of them have the label name(s) I have given them when setting them up, so I can find them in the list and then (while on edit mode in my Android quick settings) drag and drop them in my active tiles.... 

Unless you are suggesting to just drag and drop any of the available ones, regardless of the fact that they are still named `Tile Not Setup`. 🤔 I will try and do that and see if that works. If that is the case I will make an edit on the documentation to become clearer how this process works.

**Comment by prantzos** [comment]
It works! 🎉
Now I can get back to setting up my HA tiles 😃

![Screenshot_20220123-162028](https://user-images.githubusercontent.com/8318629/150685894-a3d9386b-c304-4f60-ac38-13e754fc8741.png)


@dshokouhi thank you for getting back to me so fast!! 

I think it is quite unclear how this works though... 🤔

If the name of the tile was to be updated only after the tile is dropped to the active tiles area, then this issue is not a bug. Maybe it could be improved, but this is unrelated to this issue.

If the name of the tile should have been updated before it is dropped to the active tiles area, then this might be what other people are experiencing while thinking there is an issue..

**Comment by dshokouhi** [comment]
> I think it is quite unclear how this works though... 🤔\n\nIf the name of the tile was to be updated only after the tile is dropped to the active tiles area, then this issue is not a bug. Maybe it could be improved, but this is unrelated to this issue.

The name can only be updated when it's added and in view of the user. That's an Android thing.

**Comment by prantzos** [comment]
Ah I see! 👍 Thank you for clarifying that @dshokouhi
